### PR TITLE
add js and raiwidget coverage

### PR DESCRIPTION
**Pull request by xuke444**
Signed-off-by: Ke xu <xuke@microsoft.com>

## Areas changed

npm packages changed:

- [ ] responsibleai/causality
- [ ] responsibleai/core-ui
- [ ] responsibleai/counterfactuals
- [ ] responsibleai/dataset-explorer
- [ ] responsibleai/fairness
- [ ] responsibleai/interpret
- [ ] responsibleai/localization
- [ ] responsibleai/mlchartlib
- [ ] responsibleai/model-assessment

Python packages changed:

- [ ] raiwidgets
- [ ] responsibleai
- [ ] erroranalysis
- [ ] rai_core_flask

## Tests

- [ ] No new tests required.
- [ ] New tests for the added feature are part of this PR.
- [ ] I validated the changes manually.

## Documentation:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by tongyu-microsoft — APPROVED** [review]

File: .github/workflows/CI-python.yml [hunk 1]
```diff
@@ -94,19 +93,13 @@ jobs:
...
-        working-directory: ${{ matrix.packageDirectory }}
-
       - name: Upload code coverage results
         uses: actions/upload-artifact@v2
         with:
           name: ${{ matrix.packageDirectory }}-code-coverage-results
           path: ${{ matrix.packageDirectory }}/htmlcov
         # Use always() to always run this step to publish test results when there are test failures
-        if: ${{ always() && matrix.packageDirectory != 'raiwidgets'}}
+        if: ${{ always()}}
```
Comment (romanlutz) [inline .github/workflows/CI-python.yml:102]: This seems like it'll always evaluate to true so you can probably remove the line

File: .github/workflows/CI-python.yml [hunk 2]
```diff
@@ -119,4 +112,4 @@ jobs:
           name: codecov-umbrella
           path_to_write_report: ./coverage/codecov_report.txt
           verbose: true
-        if: ${{ matrix.packageDirectory != 'raiwidgets'}}
+        if: ${{ always()}}
```
Comment (romanlutz) [inline .github/workflows/CI-python.yml:115]: This seems like it'll always evaluate to true so you can probably remove the line

**Review by romanlutz — APPROVED** [review]

File: .github/workflows/CI-python.yml (hunk 1 above)
Comment (xuke444) [reply .github/workflows/CI-python.yml:102]: the difference is with "always" this step will always execute regardless whether previous steps successes or failed.
So we can always see the coverage even test failed.

**Review by imatiach-msft — APPROVED** [review]

File: raiwidgets/raiwidgets/error_analysis_dashboard.py [hunk 3]
```diff
@@ -95,35 +95,6 @@ class ErrorAnalysisDashboard(Dashboard):
         Specify less than 10k points for optimal performance.
     :type sample_dataset: pd.DataFrame or numpy.ndarray or list[][]
 
-    :Example:
-
-    Run simple view of error analysis with just predictions and true labels
```
Comment (imatiach-msft) [inline raiwidgets/raiwidgets/error_analysis_dashboard.py:100]: why are these examples being removed?  they are shown in the docs page
https://erroranalysis.ai/raiwidgets.html
image

I think these are good examples to have?

File: .github/workflows/Ci-raiwigets-python-typescript.yml [hunk 4]
```diff
@@ -77,5 +77,27 @@ jobs:
 
       - name: Run tests
         run: |
-          pytest --durations=10
+          pytest --durations=10 --doctest-modules --junitxml=junit/test-results.xml --cov=${{ matrix.packageDirectory }} --cov-report=xml --cov-report=html
```
Comment (imatiach-msft) [inline .github/workflows/Ci-raiwigets-python-typescript.yml:80]: I think you can add examples back if you remove the "--doctest-modules" parameter here

**Pull request merged**